src/burnchains/bitcoin/bits.rs

```diff
@@ -54,107 +67,33 @@ pub fn parse_script<'a>(script: &'a Script) -> Vec<Instruction<'a>> {
     return ret;
```

Comment:
`parse_script` can be replaced with a single `.collect()`:

```rust
pub fn parse_script<'a>(script: &'a Script) -> Vec<Instruction<'a>> {
    script.iter(false).collect()
}
```

Reply:
Addressed in 9f86c7a4